sources/packages/backend/apps/api/src/services/institution/institution.service.ts

```diff
@@ -795,6 +796,7 @@ export class InstitutionService extends RecordDataModelService<Institution> {
    * @param noteDescription note description.
    * @param auditUserId user that should be considered the one that is causing the changes.
    * @returns saved Note.
+   * @deprecated use {@link createInstitutionNote} instead.
```

Comment:
FMU, this has been added so that the code using `addInstitutionNote` can be refactored later to use `createInstitutionNote`?

Reply:
The first intention is to guide anyone looking to create an institution note to use the most updated method.
One day the existing ones can be refactored.